### ISSUE TITLE
MemberVarSpacingSniff – allow var on first line of class

### ISSUE THREAD

**Issue by glen-84** [squizlabs/PHP_CodeSniffer#725]
This is not allowed:

``` php
class X
{
    public $x;

    public $y;
}
```

`Expected 1 blank line before member var; 0 found`

**Comment by glen-84** [comment]
The same is true for `Squiz.WhiteSpace.FunctionSpacing` (before/after). I'd like to specify 1 line between methods, but not above the first method or below the last.

**Comment by gsherwood** [comment]
This will now be possible from version 3.1.0. To have a single blank line between member vars, but no blank line before the first member var, put this in your ruleset.xml file:
```xml
<rule ref="Squiz.WhiteSpace.MemberVarSpacing">
    <properties>
        <property name="spacingBeforeFirst" value="0"/>
    </properties>
</rule>
```
